docs/source/hyper-parameter-search.rst

```diff
@@ -179,28 +179,29 @@ tasks in the graph and only perform the fit step once for any
 parameter/data/estimator combination. For pipelines that have relatively
 expensive early steps, this can be a big win when performing a grid search.
 
-.. _hyperparameter.incremental:
+.. _hyperparameter.adaptive:
 
 
-Incremental Hyperparameter Optimization
----------------------------------------
+Adaptive Hyperparameter Optimization
+------------------------------------
 
-The second category of hyperparameter optimization uses *incremental* hyperparameter
+The second category of hyperparameter optimization uses *adaptive* hyperparameter
 optimization.
 
 .. autosummary::
-   dask_ml.model_selection.IncrementalSearch
+   dask_ml.model_selection.DecaySearchCV
 
-Broadly speaking, incremental optimization starts with a batch of models (underlying
-estimators and hyperparameter combinationms) and repeatedly calls the underlying estimator's
-``partial_fit`` method with batches of data.
+
+These model selection algorithms are *adaptive*: they decide to keep training
+estimators based off all previously collected information. As such, these
+estimators must implement ``partial_fit`` to train on batches of data.
 
 .. note::
 
    These estimators require the optional ``distributed`` library.
 
 Here's an example training on a "large" dataset (a Dask array) with the
-``IncrementalSearch``
+``DecaySearchCV``
```

Comment:
These need to be updated for the new name.

Reply:
Updated.